internal/cmd/commands/authmethodscmd/ldap_funcs.go

```diff
@@ -369,6 +389,36 @@ func extraLdapFlagHandlingFuncImpl(c *LdapCommand, _ *base.FlagSets, opts *[]aut
 		*opts = append(*opts, authmethods.WithLdapAuthMethodGroupFilter(c.flagGroupFilter))
 	}
 
+	switch c.flagDerefAliases {
+	case "":
+	case "null":
```

Comment:
Maybe 
```suggestion
	case "", "null":
```
?

Reply:
Hum... your suggestion isn't consistent with the pattern used throughout the "commands".  It works correctly, so for now I'll be consistent.